framework/include/transfers/MultiAppProjectionTransfer.h

```diff
@@ -57,6 +59,8 @@ class MultiAppProjectionTransfer : public MultiAppTransfer
   friend void assemble_l2_from(EquationSystems & es, const std::string & system_name);
   friend void assemble_l2_to(EquationSystems & es, const std::string & system_name);
 
+  NumericVector<Number> * _serialized_master_solution;
```

Comment:
Why add this as a member variable?  As far as I can tell it's only referenced in a small area surrounding a function call.  Just pass it in to that function.